src/main/java/com/myendnoteweb/utils/WaitUtils.java

```diff
@@ -0,0 +1,360 @@
+package com.myendnoteweb.utils;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.regex.Pattern;
+
+public class WaitUtils {
+    WebDriver driver = new ChromeDriver();
+    int timeout = 30;
+    WebDriverWait wait = new WebDriverWait(driver, timeout);
+
+    public boolean waitElementToBeClickable(WebElement web) {
```

Comment:
all wait methods should not return type boolean. let's return void

Reply:
Ok

Reply:
return type was changed in all the methods of the class on the type "void"